consensus/spos/bls/subroundSignature.go

```diff
@@ -460,6 +475,21 @@ func (sr *subroundSignature) sendSignatureForManagedKey(idx int, pk string) bool
 	return sr.completeSignatureSubRound(pk, shouldWaitForAllSigsAsync)
 }
 
+func (sr *subroundSignature) checkGoRoutinesThrottler(ctx context.Context) error {
+	for {
+		if sr.signatureThrottler.CanProcess() {
```

Comment:
this will cause skipping the signatures from some managed keys. Shouldn't we enqueue all signatures, and the throttler become some sort of pool? cc @AdoAdoAdo

Reply:
discussed, invalid comment